### PR TITLE
Fix the url in the "build" badge in the readme

### PR DESCRIPTION
**Pull request by jagerber48**
See https://github.com/lmfit/uncertainties/pull/194#discussion_r1517622906

Test the correct url [here on my fork](https://github.com/jagerber48/uncertainties/tree/bugfix/build_badge_url).

### CODE REVIEW

**Comment by newville** [comment]
The URL works for me, but I think the target should be to show the success or failure of the test, not the test script itself.  I would recommend a target of https://github.com/lmfit/uncertainties/actions/workflows/python-package.yml

Also: if you are editing "README.rst", perhaps the call for maintainers should be removed too ;)

**Comment by jagerber48** [comment]
Ok, I pointed the url to the actions page instead of the action source code.

I also removed the call for maintainers. Should this go in another PR though? On this specific one though, I'd like to give @lebigot a chance to veto this change though since he put the message there in the first place and I feel like I'd be happy to know he's satisfied before removing the message.

File: README.rst [hunk 1]
```diff
@@ -11,9 +11,7 @@ uncertainties
 .. image:: https://codecov.io/gh/lebigot/uncertainties/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/lebigot/uncertainties/
 .. image:: https://img.shields.io/github/actions/workflow/status/lmfit/uncertainties/python-package.yml?logo=github%20actions
-   :target: https://github.com/lmfit/uncertainties/blob/main/.github/workflows/python-package.yml
-
-**Call for maintainers**: if you want this project to keep living and are ready to maintain it (pull requests management, issue resolution…), please contact me! I am ready to share my knowledge of the code logic by participating in discussions (notably around pull requests and issues).
+   :target: https://github.com/lmfit/uncertainties/actions/workflows/python-package.yml
```
Comment (newville) [inline README.rst:14]: Thanks!

**Review by newville — APPROVED** [review]

**Comment by newville** [comment]
@jagerber48 Thanks -- this looks good to me.  If there are no objections, I'd be happy to merge within a couple of days.

**Comment by lebigot** [comment]
> I also removed the call for maintainers. Should this go in another PR though? On this specific one though, I'd like to give @lebigot a chance to veto this change though since he put the message there in the first place and I feel like I'd be happy to know he's satisfied before removing the message.

Good with me! Based on my past interactions with all of you, I consider that the right people are now at the helm, and I trust that you'll welcome other people based on the contributions that they want to make!

**Comment by newville** [comment]
@jagerber48 let's start by merging this one!

**Pull request merged**